tests/integration/test_charm.py

```diff
@@ -130,6 +138,24 @@ async def test_service_mesh_relation(ops_test: OpsTest, service_mesh_tester):
     )
 
 
+@pytest.mark.abort_on_fail
+async def test_modeloperator_rule(ops_test: OpsTest, service_mesh_tester):
+    # Ensure model is on mesh
+    await ops_test.model.applications[APP_NAME].set_config({"model-on-mesh": "true"})
+    await ops_test.track_model("off-mesh-model")
```

Comment:
Not required, but a suggestion to make `juju models` more readable
```suggestion
    await ops_test.track_model("off-mesh-model", model_name=f"{ops_test.model.name}-off-mesh-model")
```

If you do accept this, we should also change `ops_test.track_model("istio-system")` above to `    await ops_test.track_model("istio-system", model_name=f"{ops_test.model.name}-istio-system")`.  

Making these two changes imo makes the `juju models` return more readable when debugging:

```
juju models


Controller: local-microk8s

Model                           Cloud/Region              Type        Status     Units  Access  Last connection
controller                      local-microk8s/localhost  kubernetes  available  1       admin  just now
test-charm-uu4k                 local-microk8s/localhost  kubernetes  available  4       admin  8 seconds ago
test-charm-uu4k-istio-system    local-microk8s/localhost  kubernetes  available  1       admin  6 minutes ago
test-charm-uu4k-off-mesh-model  local-microk8s/localhost  kubernetes  available  -       admin  7 seconds ago
```

Reply:
Yes this is awesome!